rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1767,17 +1770,21 @@ bool spu_thread::set_ch_value(u32 ch, u32 value)
 	{
 		if (offset >= RAW_SPU_BASE_ADDR)
 		{
-			while (!ch_out_intr_mbox.try_push(value))
+			ch_out_intr_mbox.data.store(spu_channel::bit_wait | spu_channel::bit_count | value);
+
+			if (int_ctrl[2].set(SPU_INT2_STAT_MAILBOX_INT))
```

Comment:
Why it doesn't wait if no handler is executed?

Reply:
It could be simplified to wait regardless of interrupt behaviour, also this would automatically fix the setting of bit_wait flag which implies further waiting (performance issue).

Reply:
If there is no handler it acts like a regular out mailbox frrom old hw testing: stalling when trying to push data again. The reason I put this to wip is because there is more to this and this behaviour breaks a few games.